di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/handlers/QueryStringParamConstants.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.di.ipv.stub.cred.handlers;
+
+public class QueryStringParamConstants {
```

Comment:
Does it add value to have this as a separate class? Do these values just belong in `AuthorizeHandler`

Reply:
We moved them here into its own class as we think they are going to be used within the other endpoint classes